extensions/wikia/AchievementsII/AchConfig.class.php

```diff
@@ -191,66 +191,64 @@ private function fetchAll($useMasterDb = false) {
 				}
 				*/
 
-				if($row->type == BADGE_TYPE_INTRACKEDITPLUSCATEGORY)
-					$this->mInTrackEditPlusCategory[$row->id] = array('enabled' => $row->enabled, 'category' => $row->cat);
-				elseif($row->type == BADGE_TYPE_NOTINTRACKCOMMUNITYPLATINUM) {
-					$this->mNotInTrackCommunityPlatinum[$row->id] = array(
+				if( $row->type == BADGE_TYPE_INTRACKEDITPLUSCATEGORY )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)